charts/k8ssandra/values.yaml

```diff
@@ -156,30 +104,87 @@ cass-operator:
   enabled: true
 
 kube-prometheus-stack:
-  alertmanager:
+  # Disable default service monitors
+  coreDns:
     enabled: false
-
-  kubeStateMetrics:
+  kubeApiServer:
     enabled: false
-
-  grafana:
+  kubeControllerManager:
     enabled: false
-
-  nodeExporter:
+  kubeDns:
     enabled: false
-
-  kubeletService:
+  kubeEtcd:
     enabled: false
-
-  kubeControllerManager:
+  kubeProxy:
+    enabled: false
+  kubeScheduler:
+    enabled: false
+  kubeStateMetrics:
     enabled: false
-
   kubelet:
     enabled: false
+  nodeExporter:
+    enabled: false
 
-  kubeApiServer:
+  alertmanager:
     enabled: false
+    serviceMonitor:
+      selfMonitor: false
+
+  # Locks Prometheus operator to this namespace and kube-system
+  prometheusOperator:
+    namespaces:
+      releaseNamespace: true
+      additional: []
+
+    # Disable monitoring of operator
+    serviceMonitor:
+      selfMonitor: false
 
   prometheus:
-    enabled: false
+    prometheusSpec:
+      # Uncomment to listen for Prometheus traffic under a route prefix
+      # externalUrl: http://localhost:9090/prometheus
+      # routePrefix: /prometheus
+    
+    # Enable templating of ingress resources for external prometheus traffic
+    ingress:
+      enabled: false
+      paths: []
+        # Uncomment to listen for Prometheus traffic under a route prefix
+        # - /prometheus
+
+    # Disable monitoring the Prometheus instance
+    serviceMonitor:
+      selfMonitor: false
+
+    # Disable default service monitors
+    # additionalServiceMonitors: []
 
+  grafana:
```

Comment:
Don't we need the `grafana.enabled` flag to be exposed?

Reply:
It defaults to true so I didn't include it here. Users may still override this parameter in their local values.yaml if they want.